### DISCUSSION TITLE
HTTPS server

### DISCUSSION THREAD

**Discussion by javiersotoPA** [kartoza/docker-geoserver#466] [Q&A]
Hi! I am using docker_compose_nginx.yml from kartoza and I have not manage to set up an HTTPS server using my nginx. conf file. So when I create WMS/WFS layers in geoserver, I dont get a secure access to them and some platforms are rejecting the links. I am running it on an AWS instance using Ubuntu 20.04.4 LTS. How should I configure it to have HTTPS?? Many thanks

**Comment by NyakudyaA** [comment]
Are you intending to ssl over nginx or directly in Geoserver. If it's the first option then google nginx letsencrypt. If it's the second option then look at the readme about ssl
